### PR TITLE
tweets

### PR DESCRIPTION
**Pull request by kesla**
R = @iefserge
T = minor

### CODE REVIEW

File: lib/embeds/twitter.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  };
+};
+
+export default elm => {
+  return {
+    embedType,
+    parse: elm => {
+      const classes = (elm.getAttribute('class') || '').split(' ');
+      if (classes.indexOf('twitter-tweet') === -1) {
+        return;
```
Comment (iefserge) [inline lib/embeds/twitter.js]: not consistent return, I think eslint should flag those

**Comment by iefserge** [comment]
@kesla what's with the lowercases?
Minor comment otherwise looks good.

File: lib/embeds/twitter.js (hunk 1 above)
Comment (kesla) [inline lib/embeds/twitter.js]: oh, good point. Thanks!

**Comment by kesla** [comment]
@iefserge right, the lowercases should've been a separate PR - but it's just simplifying things a little. Lowercasing the tagNames once instead of in multiple places.

**Comment by iefserge** [comment]
@kesla it may affect comparisons in a few places, like when we're comparing to 'a'

**Comment by kesla** [comment]
@iefserge it'll be fine since we run `toLowerCase()` here https://github.com/micnews/html-to-article-json/pull/10/files#diff-6731036b2b2841cbf71f5f79ff8cab0cR37

**Comment by iefserge** [comment]
@kesla ok

**Pull request merged**